### ISSUE TITLE
Elements on homepage constantly resize in Chrome

### ISSUE THREAD

**Issue by achalv** [adamschwartz/magic-of-css#15]
![8qwcu](https://cloud.githubusercontent.com/assets/1014428/2950496/2fd68bd4-da1b-11e3-9210-ddd364bca14e.gif)

I'm using a 13" rMBP running OSX 10.9.2, and used Chrome 29.0.1547.57 to view the page. It works just fine on Firefox and Safari, though.

**Comment by cvrebert** [comment]
Your Chrome is pretty outdated. Latest stable version is 34.0.1847.131.

**Comment by adamschwartz** [comment]
Yeah I believe that's due to a (now-fixed) bug in the way `rem` units were handled in media queries. Closing for now, but please re-open if you see this in the latest stable version of Chrome.

**Comment by hugokrantz** [comment]
Just a note: This is still an issue in Safari (8.0.5).

**Comment by sentience** [comment]
On [Chapter 1](http://adamschwartz.co/magic-of-css/chapters/1-the-box/) I can reproduce this in Safari by tabbing around the page. In particular, when focus is in the text field on this page, the content is small, but when when I click away from that field, the content gets big again.
